### ISSUE TITLE
Vote for Kubekey's logo

### ISSUE THREAD

**Issue by FeynmanZhou** [kubesphere/kubekey#467]
Which icon you prefer? Feel free to leave your comments below:
 
- Icon 1

![image](https://user-images.githubusercontent.com/40452856/108707665-555bab00-754b-11eb-852c-461f9c1aede1.png)

- Icon 2

![image](https://user-images.githubusercontent.com/40452856/108707672-57be0500-754b-11eb-9eed-3fdd6a8452d5.png)

**Comment by Sherlock113** [comment]
I reckon the second "k" should be capitalized in both versions.

**Comment by calvinyv** [comment]
any other choices? :( I think the 2nd icon is much more matched for KubeEye...

**Comment by rayzhou2017** [comment]
I agree with @Sherlock113 and @calvinyv. The first one is too complicated. The second is more suitable for KubeEye.

**Comment by Forest-L** [comment]
I think the Logo can add a little Kubernetes element information.

**Comment by geneliu** [comment]
like the 2nd one more! :)

**Comment by LinuxSuRen** [comment]
I guess that we can close this issue against #482 and #484

**Comment by xiabai84** [comment]
Absolutely the 2nd

**Comment by FeynmanZhou** [comment]
The new logo for KubeKey is coming, how do you guys think of it?
![image](https://user-images.githubusercontent.com/40452856/127618792-2840e8c7-48c0-4a2f-a280-77e322e21211.png)
![image](https://user-images.githubusercontent.com/40452856/127618806-a078844f-f770-463a-8eb3-6558d5576969.png)
